src/main/java/Duke.java

```diff
@@ -1,10 +1,86 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
+    private final static ArrayList<Task> taskList = new ArrayList<>();
+    static String lineBreak = "-----------------";
+
     public static void main(String[] args) {
```

Comment:
Your main method feels a bit long, can consider modularizing it. According to the [textbook](https://nus-cs2113-ay2223s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-long-methods), the recommendation is to have 30 LOC.